### PR TITLE
Objc proto2 conformance test support

### PR DESCRIPTION
**Pull request by thomasvl**

### CODE REVIEW

File: conformance/conformance_objc.m [hunk 1]
```diff
@@ -63,28 +64,29 @@ static void Die(NSString *format, ...) {
...
 
     case ConformanceRequest_Payload_OneOfCase_ProtobufPayload: {
-      if ([request.messageType isEqualToString:@"protobuf_test_messages.proto3.TestAllTypesProto3"]) {
+      Class msgClass = nil;
+      if ([request.messageType isEqual:@"protobuf_test_messages.proto3.TestAllTypesProto3"]) {
+        msgClass = [Proto3TestAllTypesProto3 class];
+      } else if ([request.messageType isEqual:@"protobuf_test_messages.proto2.TestAllTypesProto2"]) {
+        msgClass = [TestAllTypesProto2 class];
+      } else {
+        Die(@"Protobuf request doesn't have specific payload type");
```
Comment (haberman) [inline conformance/conformance_objc.m]: Error message seems misleading; should be `"unknown message type: " + messageType`

File: conformance/conformance_objc.m [hunk 2]
```diff
@@ -63,28 +64,29 @@ static void Die(NSString *format, ...) {
...
+      }
+      if (msgClass) {
```
Comment (haberman) [inline conformance/conformance_objc.m]: How can this be nil? You called `Die` in the only path that doesn't set this.

Comment (thomasvl) [inline conformance/conformance_objc.m]: Correct, tweaked.

File: conformance/conformance_objc.m (hunk 1 above)
Comment (thomasvl) [inline conformance/conformance_objc.m]: Yup, that was the message from before.  Tweaked.

**Comment by BSBandme** [comment]
Sorry that I just saw your request. So we need to update the error message for all the other language?

**Pull request merged**